modules/caddyhttp/fileserver/browsetplcontext.go

```diff
@@ -230,13 +241,14 @@ type crumb struct {
 // fileInfo contains serializable information
 // about a file or directory.
 type fileInfo struct {
-	Name      string      `json:"name"`
-	Size      int64       `json:"size"`
-	URL       string      `json:"url"`
-	ModTime   time.Time   `json:"mod_time"`
-	Mode      os.FileMode `json:"mode"`
-	IsDir     bool        `json:"is_dir"`
-	IsSymlink bool        `json:"is_symlink"`
+	Name        string      `json:"name"`
+	Size        int64       `json:"size"`
+	URL         string      `json:"url"`
+	ModTime     time.Time   `json:"mod_time"`
+	Mode        os.FileMode `json:"mode"`
+	IsDir       bool        `json:"is_dir"`
+	IsSymlink   bool        `json:"is_symlink"`
+	SymlinkPath []string    `json:"symlink_path"`
```

Comment:
Ok I guess my last question is why this is a slice? I think it could just be a string yeah?

Reply:
Isn't the first element just the Name? And the only thing we need to point to is its target (this new field)?

Reply:
You are absolutely right. I just overcomplicated it 😅 . Should be good to go now. Thank you for your feedback Matt!